### PR TITLE
Pull request review Input component

### PR DESCRIPTION
**Pull request by apsuhos**

### CODE REVIEW

File: src/styles/globalStyles.js [hunk 1]
```diff
@@ -203,4 +203,29 @@ img {
   font-style: italic;
   vertical-align: middle;
 }
+input {
```
Comment (nikoskleidis) [inline src/styles/globalStyles.js:206]: @apsuhos you can add these styles inside the InputCls class

File: src/components/Input/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+function Input({ className, onChange, label, value, errorMessage, success, errorForm, disabled, search }) {
+  const [inputValue, setInputValue] = useState("")
+  const [isDisabled, setIsDisabled] = useState(false)
+  const [error, setError] = useState(false)
+
+  errorMessage = "We are sorry, an error occured. Please try again."
+
+  return (
+    <div className={cx(InputCls, { success, errorForm, disabled, search }, className)}>
```
Comment (nikoskleidis) [inline src/components/Input/index.js:23]: errorForm should not be a property of the Input component. It should be assigned only when there is an errorMessage property to the component. eg
```suggestion
    <div className={cx(InputCls, { success, errorForm: Boolean(errorMessage), disabled, search }, className)}>
```

File: src/styles/globalStyles.js (hunk 1 above)
Comment (apsuhos) [reply src/styles/globalStyles.js]: Done

File: src/components/Input/index.js (hunk 2 above)
Comment (spiropoulos94) [reply src/components/Input/index.js]: image

why am i not allowed to commit your suggestion?

**Comment by nikoskleidis** [comment]
Also please get the latest changes from the develop branch and merge them into yours in order to resolve the conflicts

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        },
+        "ecmaVersion": 11,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+    }
+}
```
Comment (nikoskleidis) [inline .eslintrc.json:22]: Please remove this file. We currently use the .eslintrc.js for the eslint rules

**Comment by spiropoulos94** [comment]
image
why am i not allowed to merge? i already did it in the morning just fine.. why doesnt it work now?

**Comment by nikoskleidis** [comment]
> image
> why am i not allowed to merge? i already did it in the morning just fine.. why doesnt it work now?

Sorry but I have restricted merge ability only to myself!

**Pull request merged**